Dockerfile

```diff
@@ -0,0 +1,36 @@
+# The base image to start from. A image hash can be used here to pick some
+# intermediate image.
+FROM ubuntu:xenial
+#FROM a2799408863
+
+
+RUN apt-get update && apt-get install -y \
+    autoconf automake libtool libleptonica-dev libicu-dev libpango1.0-dev libcairo2-dev cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev python-dev python-numpy libjpeg-dev libpng-dev libtiff-dev libjasper-dev libdc1394-22-dev virtualenvwrapper liblog4cplus-dev libcurl4-openssl-dev libtiff5-dev gcc make ca-certificates autoconf-archive libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev \
+    wget unzip python3-pip python3-mysql.connector libtesseract3 #libopencv-dev
+RUN apt-get install -y openjdk-8-jdk default-jdk
+RUN apt-get install -y libgtk-3-dev libv4l-dev
+#RUN cd /usr/include/linux && ln -s ../libv4l1-videodev.h videodev.h
+
+# Download the leptonica if not already present.
+RUN test -f /tmp/leptonica-1.76.0.tar.gz || wget http://www.leptonica.org/source/leptonica-1.76.0.tar.gz -O /tmp/leptonica-1.76.0.tar.gz
+
+RUN cd /tmp/ && tar xvf leptonica-1.76.0.tar.gz --no-same-owner && cd leptonica-1.76.0 && \
+    ./configure && \
+    make -j2 && \
+    make install
+
+#RUN cd /usr/src && git clone https://github.com/tesseract-ocr/tesseract.git && \
```

Comment:
This as well seemed not needed. I did add libtesseract3 as an apt-get dependency on line #9 IIRC this is the reason I removed it.